src/main/java/com/internet/shop/controllers/item/GetAllItemsController.java

```diff
@@ -0,0 +1,29 @@
+package com.internet.shop.controllers.item;
+
+import com.internet.shop.library.Injector;
+import com.internet.shop.model.Product;
+import com.internet.shop.service.interfaces.ProductService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/items/all")
+public class GetAllItemsController extends HttpServlet {
```

Comment:
```suggestion
public class GetAllProductsController extends HttpServlet {
```
fix naming consistency everywhere